### DISCUSSION TITLE
Ibis - Migrating Ibis documentation to Quarto

### DISCUSSION THREAD

**Discussion by giscus** [ibis-project/ibis#6873] [Q&A] [answered]
# Ibis - Migrating Ibis documentation to Quarto

https://ibisdocs.z5.web.core.windows.net/posts/website-to-quarto/

**Comment by lostmygithubaccount** [comment, marked as answer]
https://ibisdocs.z5.web.core.windows.net/posts/website-to-quarto/

**Reply by lostmygithubaccount** [reply]
this was a test message -- can remove/delete
